docs/proposals/label-associated-quota.md

```diff
@@ -0,0 +1,72 @@
+# Label Constrained Quota
+
+## Problem
+Cluster-administrators want to allot a total quota cap to a particular entity (users to start with) and allow that entity to allocate the quota amongst "his" projects as he sees fit.
+Our mechanism for relating entities is label selection, so we can create a `ResourceQuotaAllocation` object at the cluster scope that applies allotment totals against `ResourceQuota`
+objects.  By associating projects to owners using structured labels, it becomes possible to allocate quota the projects "owned" by a given user.
+
+
+## Types
+```go
+type ResourceQuotaAllocation struct{
```

Comment:
> Can I have multiple allocations that select the same set of projects? If so, what does that mean?

Yes, you can.  It works congruently to multiple `ResourceQuota` objects in a namespace.  Smallest allowed size wins.